sample_files/.gitignore

```diff
@@ -55,5 +73,5 @@ docs/_build/
 *~
 *.swp
 
-# OSX Files
-*.DS_Store
+# OCA rules
+!static/lib/
```

Comment:
Whoah whoah whoah whoah whoah. This is a thing?!?!?!?!

Welp TIL

Reply:
What this means?

Reply:
Gitignore supports negation patterns, which is what this is. In web repos, we have a valid `static/lib` folder that is typically excluded by the [`lib/` exclusion for Python distribution](https://github.com/Eficent/maintainer-quality-tools/blob/cbc551372084c82ab81cdb0b3cea806e79115f9f/sample_files/.gitignore#L19). Adding this negation after the linked declaration allows for both to exist in harmony.

If you take a look at the [`OCA/web` repo gitignore](https://github.com/OCA/web/blob/11.0/.gitignore), you'll notice we had to actually remove the `lib/` declaration. @mreficent suggestion fixes that.

So yeah, that's my TIL (today I learned) 🚀

Reply:
Indeed!